### PR TITLE
FIX: http.request.body doesn't exists

### PR DESCRIPTION
**Pull request by m1keil**
Also tweak error part of the errorMiddlewere

### CODE REVIEW

**Comment by hidday** [comment]
Please add label

File: utils.js [hunk 1]
```diff
@@ -60,7 +60,9 @@ function ecsMeta(req, res) {
     }
 
     if (req.headers['content-length']) {
-      meta.http.request.body.bytes = req.headers['content-length'];
+      meta.http.request.body = {
```
Comment (nir0s) [inline utils.js:63]: Um... but that will completely override the. body, no?

File: utils.js [hunk 2]
```diff
@@ -83,8 +85,9 @@ function ecsMeta(req, res) {
     if (port) meta.url.port = Number(port);
   }
 
-
```
Comment (nir0s) [inline utils.js:86]: Capitalize, please.

File: utils.js (hunk 1 above)
Comment (m1keil) [reply utils.js:63]: mm yeah, but http.request.body doesn't exists.

**Pull request merged**